hub/compute/tests/test_ray.py

```diff
@@ -13,7 +13,7 @@
 
 my_schema = {
     "image": Tensor((28, 28, 4), "int32", (28, 28, 4)),
-    "label": "<U20",
+    "label": Text((None,), "int64", (20,)),
```

Comment:
Can you just write Text()?

It's unclear why the user should specify int64. 

Maybe just an optional argument max_length?

Reply:
Yes, we can set the default to int64 and avoid dtype altogether for Text

Reply:
yeah, that's great! as much simpler as better for the user :)